docs/resources/cloud_discovery_provider.md

```diff
@@ -0,0 +1,293 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "bloxone_cloud_discovery_provider Resource - terraform-provider-bloxone"
+subcategory: "Cloud Discovery Providers"
+description: |-
+  Manages Cloud Discovery Providers.
+---
+
+# bloxone_cloud_discovery_provider (Resource)
+
+Manages Cloud Discovery Providers.
+
+## Example Usage
+
+```terraform
+resource "bloxone_dns_view" "example" {
+  name = "example_dns_view"
+}
+
+
+resource "bloxone_cloud_discovery_provider" "example_aws" {
```

Comment:
nit: would be nice if we could keep the examples for each service in a different block. You will have to override the template for this.

Reply:
Done